### ISSUE TITLE
1.3.30 to 1.5.6 upgrade issue with Trial Balance reports 

### ISSUE THREAD

**Issue by billott323** [ledgersmb/LedgerSMB#2797]
When selecting an account from the trial balance screen, The balance column in the resulting report is not ordered, if no start date was selected.  Here is an example

```
04/25/2017 	399 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	9030.27
04/25/2017 	401 	Peak City Holdings 	0 	2161.92 	other xfer to PCH 	1060 	-34634.87
04/28/2017 	871 	Darren Hoch 	400.00 	0 	check 5123 	1060 	26721.51
04/28/2017 	874 	ModWraps 	832.50 	0 	check 1066 	1060 	-81783.42
04/28/2017 	882 	Triangle Risk Management 	175.00 	0 	check 2537 	1060 	9149.45
05/02/2017 	232 	Equity Payout 	0 	3400.00 		1060 	14012.09
05/02/2017 	232 	Equity Payout 	0 	3400.00 		1060 	1575.00
05/03/2017 	877 	Lori Aveni, CPA, PLLC 	725.00 	0 	check 2201 	1060 	-82881.52
05/03/2017 	884 	Compass Counseling 	600.00 	0 	check 1762 	1060 	-87515.75
			907404.38 	892161.86 	
``` 

Here it is with a start date...

```

Report Name:
General Ledger Report
Company:
SalemStreetN
Start Date:
2017-04-25
End Date:
Account Number:
1060
Reference:
Source:
Date 	Reference 	Description 	Debits 	Credits 	Source 	Account No. 	Balance
04/25/2017 	378 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-23075.02
04/25/2017 	379 	Peak City Holdings 	0 	274.58 	other Xfer to PCH 	1060 	-22800.44
04/25/2017 	381 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-22725.44
04/25/2017 	382 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-22650.44
04/25/2017 	383 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-22575.44
04/25/2017 	384 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-22500.44
04/25/2017 	385 	Peak City Holdings 	0 	113.25 	other Xfer to PCH 	1060 	-22387.19
04/25/2017 	386 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-22312.19
04/25/2017 	388 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-22237.19
04/25/2017 	389 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-22162.19
04/25/2017 	390 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-22087.19
04/25/2017 	392 	Peak City Holdings 	0 	202.00 	other Xfer to PCH 	1060 	-21885.19
04/25/2017 	394 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-21810.19
04/25/2017 	396 	Peak City Holdings 	0 	113.25 	other Xfer to PCH 	1060 	-21696.94
04/25/2017 	397 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-21621.94
04/25/2017 	398 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-21546.94
04/25/2017 	399 	Peak City Holdings 	0 	75.00 	other Xfer to PCH 	1060 	-21471.94
04/25/2017 	401 	Peak City Holdings 	0 	2161.92 	other xfer to PCH 	1060 	-19310.02
04/28/2017 	871 	Darren Hoch 	400.00 	0 	check 5123 	1060 	-19710.02
04/28/2017 	874 	ModWraps 	832.50 	0 	check 1066 	1060 	-20542.52
04/28/2017 	882 	Triangle Risk Management 	175.00 	0 	check 2537 	1060 	-20717.52
05/02/2017 	232 	Equity Payout 	0 	3400.00 		1060 	-17317.52
05/02/2017 	232 	Equity Payout 	0 	3400.00 		1060 	-13917.52
05/03/2017 	877 	Lori Aveni, CPA, PLLC 	725.00 	0 	check 2201 	1060 	-14642.52
05/03/2017 	884 	Compass Counseling 	600.00 	0 	check 1762 	1060 	-15242.52
			2732.50 	10640.00 			

[permalink]  [CSV] 

```

**Comment by billott323** [comment]
I have good news and bad news. The good news is. I have finally moved to my new server. The last step in the process was to start from the beginning. I re-installed the OS, then ran all of the install scripts, in order.  Everything related to LedgerSMB is working. I had to assign it to https://www.theotts.org instead of https://www.theotts.org/ledgersmb since I could not get the latter to work with starman.
